Makefile

```diff
@@ -17,3 +17,4 @@ wards              :; ./scripts/wards.sh $(target)
 time               :; ./scripts/time.sh date="$(date)" stamp="$(stamp)"
 exec-hash          :; ./scripts/hash-exec-copy.sh $(url)
 fund-pause-proxy   :; ./scripts/fund-pause-proxy.sh
+rate               :; ./scripts/rate.sh $(pct)
```

Comment:
actually, it might be more appropriate `rates` as you suggested, as the script can handle multiple rates either by listing all rates or computing specific ones

Reply:
done in 3dc5123